### PR TITLE
Enable deploying Presidio services on a single App Service Plan

### PR DESCRIPTION
**Pull request by balteravishay**
This PR enables users to deploy presidio services on either a single App Service Plan (sku set to P2V3 to support both analyzer and image redactor), or on 3 different plans with different skus.
The default is now a single App Service Plan and takes affect on all deploy to azure scenarios and buttons.

### CODE REVIEW

File: docs/samples/deployments/app-service/presidio-services.json [hunk 1]
```diff
@@ -204,7 +277,7 @@
                         "value": "[parameters('imagesArePublic')]"
                     },
                     "image": {
-                        "value": "[parameters('anonymizerImage')]"
+                        "value": "[parameters('imageRedactorImage')]"
```
Comment (navalev) [inline docs/samples/deployments/app-service/presidio-services.json:280]: seems that the imageRedactor deployment is done twice? there's a second image-redactor deployment later in the file

File: docs/samples/deployments/app-service/presidio-services.json [hunk 2]
```diff
@@ -135,12 +142,69 @@
...
+                    }
+                }
+            }
+        },
         {
             "type": "Microsoft.Resources/deployments",
             "apiVersion": "2019-10-01",
             "name": "presidio-analyzer",
+            "dependsOn": [
+                "[resourceId('Microsoft.Resources/deployments', 'presidio-anonymizer')]"
```
Comment (navalev) [inline docs/samples/deployments/app-service/presidio-services.json:206]: what happens if the anonymizer will not be deployed?

Comment (balteravishay) [reply docs/samples/deployments/app-service/presidio-services.json:206]: nice catch!

Comment (balteravishay) [reply docs/samples/deployments/app-service/presidio-services.json:206]: didn't need the dependsOn. ARM knows how to handle it.

File: docs/samples/deployments/app-service/presidio-services.json (hunk 1 above)
Comment (balteravishay) [reply docs/samples/deployments/app-service/presidio-services.json]: fixed

**Review by navalev — DISMISSED** [review]

File: docs/samples/deployments/app-service/presidio-services.json [hunk 3]
```diff
@@ -229,15 +251,18 @@
             "properties": {
                 "mode": "Incremental",
                 "templateLink": {
-                    "uri": "https://raw.githubusercontent.com/microsoft/presidio/main/docs/samples/deployments/app-service/presidio-app-service.json",
+                    "uri": "https://raw.githubusercontent.com/microsoft/presidio/feature/app-services-single-plan/docs/samples/deployments/app-service/presidio-app-service.json",
```
Comment (SharonHart) [inline docs/samples/deployments/app-service/presidio-services.json:254]: this needs to be changed before merging right?

**Review by SharonHart — APPROVED** [review]

**Review by omri374 — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

**Pull request merged**